pkg/daemon/ceph/client/mirror.go

```diff
@@ -119,6 +120,20 @@ func CreateRBDMirrorBootstrapPeer(context *clusterd.Context, clusterInfo *Cluste
 func enablePoolMirroring(context *clusterd.Context, clusterInfo *ClusterInfo, pool cephv1.NamedPoolSpec) error {
 	logger.Infof("enabling mirroring type %q for pool %q", pool.Mirroring.Mode, pool.Name)
 
+	if pool.Mirroring.Mode == "init-only" && !clusterInfo.CephVersion.IsAtLeastTentacle() {
+		return fmt.Errorf("ceph version %q does not support mirroring mode %s, minimum ceph version required is %q", clusterInfo.CephVersion.String(), pool.Mirroring.Mode, version.Tentacle.String())
+	}
+
+	mirrorInfo, err := GetPoolMirroringInfo(context, clusterInfo, pool.Name)
+	if err != nil {
+		return errors.Wrapf(err, "failed to get mirroring info for the radosnamespace %q", pool.Name)
+	}
+
+	if mirrorInfo.Mode != "disabled" {
+		logger.Infof("mirroring is already enabled on the pool %s", pool.Name)
+		return nil
+	}
```

Comment:
If user wants to change the mirroring mode, i think this check will block it as mirroring is already enabled , can yo u please check?

Reply:
with then condition, the problem should not occur